.github/renovate.json5

```diff
@@ -1,7 +1,20 @@
 {
-  enabledManagers: ["github-actions", "gomod"],
-  postUpdateOptions: [
-    "gomodTidy", // go mod tidy
-    "gomodUpdateImportPaths",
+  enabledManagers: ["github-actions", "gomod", "regex"],
+  postUpdateOptions: ["gomodTidy", "gomodUpdateImportPaths"],
+  regexManagers: [
+    {
+      fileMatch: ["Dockerfile$"],
```

Comment:
this will only catch files where `Dockerfile` is at the end of the file name. Do we want to restrict it to those?

```
something-Dockerfile // will match
Dockerfile // will match
Dockerfile-gci // no match
```

Reply:
The goal is just to catch the subo Dockerfiles so that any `*_VERSION=` string could be auto updated.